cmd/kops/update_cluster.go

```diff
@@ -64,8 +64,10 @@ func NewCmdUpdateCluster(f *util.Factory, out io.Writer) *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:   "cluster",
-		Short: "Update cluster",
-		Long:  `Updates a k8s cluster.`,
+		Short: "Execute the creation or modification of a cloud or cluster resources.",
+		Long: `Execute the creation cloud resources to match the current cluster specification. If the cluster or cloud resources already
+		exist this command may modify those resources.  If nodes need updating such as during a Kubernetes upgrade, a rolling-update may
```

Comment:
Can we do a new para, then, "If nodes need updating (for example to upgrade Kubernetes) a rolling-update may be required after running update"

Otherwise it implies that update does that rolling update, IMO